src/envs/behavior.py

```diff
@@ -441,26 +443,38 @@ def make_behavior_option(name: str, types: Sequence[Type], params_space: Box,
     def _policy(state: State, memory: Dict, _objects: Sequence[Object],
                 _params: Array) -> Action:
         assert "has_terminated" in memory
-        assert ("controller" in memory and memory["controller"] is not None
-                )  # must call initiable() first, and it must return True
+        assert memory.get(
+            "policy_controller"
+        ) is not None  # must call initiable() first, and it must return True
```

Comment:
formatting is a bit weird, thanks yapf -_- can you move this comment to its own line directly preceding the line of code, so that  `assert memory.get("policy_controller") is not None` fits in one line?